packages/core/src/index.tsx

```diff
@@ -17,6 +18,7 @@ export {
     DdRum,
     DdSdkReactNativeConfiguration,
     DdSdkReactNative,
+    InternalLog,
```

Comment:
since we are exporting it now, let's add the notes to the class description that it shouldn't be used by the customers